index.js

```diff
@@ -150,10 +150,10 @@ module.exports = function(version, _options) {
             if (name && ref && name !== ref && !wayMotorway) {
                 var phrase = instructions[language][version].phrase['name and ref'] ||
                     instructions.en[version].phrase['name and ref'];
-                wayName = this.tokenize(phrase, {
+                wayName = this.tokenize(language, phrase, {
```

Comment:
Argument order got mixed up in https://github.com/Project-OSRM/osrm-text-instructions/pull/148. Strange tests passed then.

Reply:
Sorry, I missed this when committing 5f4f0734b8ff92b34263d05eae55e251698b909d at the same time for #152. Probably best to keep this PR focused on roundabouts.